vbezbakh/docker/vbezbakh:devopscampsummer2019_vbezbakh_docker_example2/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM ubuntu:14.04
+RUN apt-get update
+RUN apt-get -y install cowsay
```

Comment:
Better to put the commands: 1) apt-get update, 2) apt-get install and 3) (additional) rm -rf /var/lib/apt/lists/* into one RUN commands chain.